### PR TITLE
HOWTORELEASE.md: Describe Github label workflow and fix commands

### PR DESCRIPTION
**Pull request by ricardoasmarques**
Improvements inspired on https://github.com/SUSE/sesdev/pull/109

Signed-off-by: Ricardo Marques <rimarques@suse.com>

### CODE REVIEW

File: HOWTORELEASE.md [hunk 1]
```diff
@@ -2,15 +2,22 @@
...
-    1. Update `CHANGELOG.md` with all important changes introduced since previous version.
-        - Create a new section `[<version_number>] <date YYYY-MM-DD>` and move all entries
-          from the `[Unreleased]` section to the new section.
-        - Make sure all github issues resolved in this release are referenced in the changelog.
-        - Update the links at the bottom of the file.
-    2. Update version number in `ceph-bootstrap.spec` to `Version: <version_number>`.
-2. Create an annotated tag for the above commit: `git tag -s -a v<version_number>`.
+1. Make sure you are working on the current tip of the master branch.
+2. Make sure the merged PRs of all important changes have the "Add To Changelog" label:
+   https://github.com/SUSE/ceph-bootstrap/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Amerged+label%3A%22Add+To+Changelog%22+
```
Comment (smithfarm) [inline HOWTORELEASE.md:7]: `s/Add+To+Changelog/AddToChangelog/` ?

File: HOWTORELEASE.md [hunk 2]
```diff
@@ -2,15 +2,22 @@
...
     - The message should be `version <version_number>`.
-    - Using `git show`, review the commit message of the annotated tag.
+    - Using `git show v<version_number>`, review the commit message of the annotated tag.
       It should say: `version <version_number>`.
-3. Push commit and tag to github repo: `git push <remote> master --tags`
\ No newline at end of file
+7. Push commit and tag to github repo: `git push <remote> master --tags`
+8. Remove the "Add To Changelog" labels from all the merged PRs
+9. Verify that no merged PRs have "Add To Changelog" label:
+   https://github.com/SUSE/ceph-bootstrap/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Amerged+label%3A%22Add+To+Changelog%22+
```
Comment (smithfarm) [inline HOWTORELEASE.md:23]: `s/Add+To+Changelog/AddToChangelog/` ?

File: HOWTORELEASE.md [hunk 3]
```diff
@@ -2,15 +2,22 @@
...
+   modifications to `CHANGELOG.md` made in the previous two steps.
+6. Create an annotated tag for the above commit: `git tag -s -a v<version_number> -m"version <version_number>"`.
     - The message should be `version <version_number>`.
-    - Using `git show`, review the commit message of the annotated tag.
+    - Using `git show v<version_number>`, review the commit message of the annotated tag.
       It should say: `version <version_number>`.
-3. Push commit and tag to github repo: `git push <remote> master --tags`
\ No newline at end of file
+7. Push commit and tag to github repo: `git push <remote> master --tags`
+8. Remove the "Add To Changelog" labels from all the merged PRs
```
Comment (smithfarm) [inline HOWTORELEASE.md:21]: `s/Add To Changelog/AddToChangelog/` ?

File: HOWTORELEASE.md [hunk 4]
```diff
@@ -2,15 +2,22 @@
...
-1. Create a commit with title `Bump to v<version_number>` with the following changes:
-    1. Update `CHANGELOG.md` with all important changes introduced since previous version.
-        - Create a new section `[<version_number>] <date YYYY-MM-DD>` and move all entries
-          from the `[Unreleased]` section to the new section.
-        - Make sure all github issues resolved in this release are referenced in the changelog.
-        - Update the links at the bottom of the file.
-    2. Update version number in `ceph-bootstrap.spec` to `Version: <version_number>`.
-2. Create an annotated tag for the above commit: `git tag -s -a v<version_number>`.
+1. Make sure you are working on the current tip of the master branch.
+2. Make sure the merged PRs of all important changes have the "Add To Changelog" label:
```
Comment (smithfarm) [inline HOWTORELEASE.md:6]: `s/Add To Changelog/AddToChangelog/` ?

**Review by smithfarm — APPROVED** [review]
In ceph-bootstrap, it looks like the label has no spaces in it: `AddToChangelog`

Other than that, it LGTM

**Comment by ricardoasmarques** [comment]
> In ceph-bootstrap, it looks like the label has no spaces in it: `AddToChangelog`
> 
> Other than that, it LGTM

I've renamed the label on Github for consistency. Thanks.

**Pull request merged**